lib/still/compiler/view_helpers.ex

```diff
@@ -18,19 +18,25 @@ defmodule Still.Compiler.ViewHelpers do
       @env unquote(variables)
 
       def include(file) do
-        include(file, %{})
+        include(file, %{}, [])
       end
 
-      def include(file, variables) when is_list(variables) do
-        include(file, variables |> Enum.into(%{}))
+      def include(file, variables) do
+        include(file, variables, [])
       end
 
-      def include(file, variables) do
-        with pid when not is_nil(pid) <-
-               Incremental.Registry.get_or_create_file_process(file),
-             %SourceFile{content: content} <-
-               Incremental.Node.render(pid, variables, @env[:input_file]) do
-          Incremental.Node.add_subscription(self(), file)
+      def include(file, variables, opts) when is_list(variables) do
+        include(file, variables |> Enum.into(%{}), opts)
+      end
+
+      def include(file, variables, opts) do
```

Comment:
Couldn't these all be condensed into:

```elixir
def include(file, variables, opts \\ []) when is_list(variables) do
  # ...
end

def include(file, variables \\ %{}, opts \\ []) do
  # ...
end
```

Reply:
If I'm not mistaken, that doesn't work, default values cannot be defined more than once.

Reply:
You can do

```elixir
def include(file, variables \\ %{}, opts \\[])

def include(file, variables, opts) when is_list(variables) do
  # ...
end

def include(file, variables, opts) do
  # ...
end
```